test-suite/Makefile

```
@@ -112,7 +112,7 @@ INTERACTIVE := interactive
 UNIT_TESTS := unit-tests
 VSUBSYSTEMS := prerequisite success failure $(BUGS) output output-coqtop \
   output-modulo-time $(INTERACTIVE) micromega $(COMPLEXITY) modules stm \
-  coqdoc ssr primitive/uint63 primitive/float ltac2
+  coqdoc ssr primitive/uint63 primitive/float primitive/arrays ltac2
```

perhaps
```suggestion
  coqdoc ssr $(wildcard primitive/*) ltac2
```
would be more future proof

---

Done, thanks.